src/js/util.js

```diff
@@ -50,7 +48,7 @@ var Util;
          * See #591
          */
         isMetaCtrlKey: function (event) {
-            if ((this.isMac && event.metaKey) || (!this.isMac && event.ctrlKey)) {
+            if ((Util.isMac && event.metaKey) || (!Util.isMac && event.ctrlKey)) {
```

Comment:
`Util` isn't yet created so you should keep `this`, nope? Like you did in `Selection`.